exercises/diffie-hellman/diffie-hellman.spec.js

```diff
@@ -0,0 +1,67 @@
+import DiffieHellman from './diffie-hellman';
+
+describe('diffie-hellman', () => {
+  const p = 23;
+  const g = 5;
+  let diffieHellman = new DiffieHellman(p, g);
+
+  const alicePrivateKey = 6;
+  const alicePublicKey = 8;
+
+  const bobPrivateKey = 15;
+  const bobPublicKey = 19;
+
+  test('throws an error if the constructor arguments are out of range', () => {
+
+    expect(() => {
+      new DiffieHellman(0, 9999)
+    }).toThrow();
+
+  });
+
+  test('throws an error if the constructor arguments are not prime', () => {
+
+    expect(() => {
+      new DiffieHellman(10, 13)
+    }).toThrow();
+
+  });
+
+  test('throws an error if private key >= p-1', () => {
```

Comment:
@rchavarria I'm convinced.  I'll make those changes 😁

Also, I thought it was interesting, some of the other test suites check big primes (dozens of digits), or have their Diffie-Hellman class generate random private keys and test for randomness.  Seems like those tests are not needed for this sort of exercise.